hack/jenkins/benchmark-dockerized.sh

```diff
@@ -52,4 +52,5 @@ cd /go/src/k8s.io/kubernetes
 
 # Run the benchmark tests and pretty-print the results into a separate file.
 make test-integration WHAT="$*" KUBE_TEST_ARGS="-run='XXX' -bench=. -benchmem" \
+  | tee >(go run test/integration/framework/jsonify_benchmark.go ${ARTIFACTS_DIR}/BenchmarkResults_benchmark_$(date -u +%Y-%m-%dT%H:%M:%SZ).txt)
```

Comment:
Maybe change the order of the tees, given that the latter one is more safer and you may panic on this one?

Reply:
I might be understanding sth wrong here.